opts/envfile.go

```diff
@@ -23,14 +23,14 @@ func ParseEnvFile(filename string) ([]string, error) {
 		line := scanner.Text()
 		// line is not empty, and not starting with '#'
 		if len(line) > 0 && !strings.HasPrefix(line, "#") {
-			if strings.Contains(line, "=") {
-				data := strings.SplitN(line, "=", 2)
-
-				// trim the front of a variable, but nothing else
-				variable := strings.TrimLeft(data[0], whiteSpaces)
-				if strings.ContainsAny(variable, whiteSpaces) {
-					return []string{}, ErrBadEnvVariable{fmt.Sprintf("variable '%s' has white spaces", variable)}
-				}
+			data := strings.SplitN(line, "=", 2)
+
+			// trim the front of a variable, but nothing else
+			variable := strings.TrimLeft(data[0], whiteSpaces)
+			if strings.ContainsAny(variable, whiteSpaces) {
```

Comment:
I would think it would be better to use a regex and validate the env var name against the sh definition:
Variables set by the user must have a name consisting solely of alphabetics, numerics, and underscores - the first of which must not be numeric.
I know this is old code but I noticed it another PR and we might as well fix it right.